.pre-commit-config.yaml

```diff
@@ -25,7 +25,6 @@ repos:
     - --no-sort-keys
     - '--indent=  '
   - id: check-added-large-files
-  - id: requirements-txt-fixer
```

Comment:
why remove this?

Reply:
I wanted to organize our requirements.txt a little better so we know where each dev req is coming from - this pre-commit will sort everything by alphabetical order